test/e2e/network/networking.go

```diff
@@ -46,9 +46,9 @@ var _ = SIGDescribe("Networking", func() {
 	})
 
 	It("should provide Internet connection for containers [Conformance]", func() {
-		By("Running container which tries to ping 8.8.8.8")
+		By("Running container which tries to reach www.google.com")
 		framework.ExpectNoError(
-			framework.CheckConnectivityToHost(f, "", "ping-test", "8.8.8.8", 30))
+			framework.CheckConnectivityToHost(f, "", "www.google.com", 30))
```

Comment:
Can't do this... 
We need [Conformance] to run on air-gapped environments.

Reply:
Sorry, I could not follow the meaning of "air-gapped environments".
Could you rephrase ?

Reply:
IIUC the test as it is phrased today won't work on air-gapped environments since it tries to hit 8.8.8.8.

Reply:
Several clusters do not allow traffic or resolution to the outside world.  You can make a separate test, but it CAN NOT use the keyword [Conformance]

Reply:
The original 8.8.8.8 is the global address of google DNS, that already is traffic to the outside world from clusters.

Reply:
Hmm I'll have to fix this in a separate patch.

Reply:
I think it's fundamentally a question of whether or not the test in question should be a conformance test.  I'm not sure who defines what Conformance is today.

i.e. the test in question is called `should provide Internet connection for containers [Conformance]`

Reply:
I'll open a separate issue on this.

Reply:
Yeah, I had the same question when I found this issue.
I am not sure it is necessary to ping to the internet as one of conformance tests, because ICMP could be denied in many environments as the commit message.

Oh, sorry I opened the issue for that as https://github.com/kubernetes/kubernetes/issues/52099
If conflict, I will drop mine.